src/main/java/com/kryptnostic/conductor/rpc/ConductorSparkApi.java

```diff
@@ -14,6 +14,7 @@
 import com.dataloom.data.requests.LookupEntitiesRequest;
 import com.dataloom.edm.internal.EntitySet;
 import com.dataloom.edm.internal.PropertyType;
+import com.dataloom.organization.Organization;
 import com.google.common.base.Optional;
 
 public interface ConductorSparkApi {
```

Comment:
This starting to get big-- we should break into sub-interfaces when we get the chance.